modules/future_date_validation.js

```diff
@@ -0,0 +1,24 @@
+const isDateFuture = entered_date => {
```

Comment:
I think we should add a folder titled validation inside modules, then have the 2 date validation functions in one file called date_validation. Thoughts?

Reply:
Coolio, makes sense